src/realm/object-store/shared_realm.cpp

```diff
@@ -890,6 +890,11 @@ SharedRealm Realm::freeze()
     return Realm::get_frozen_realm(std::move(config), version);
 }
 
+SharedRealm Realm::thaw()
+{
+    return Realm::get_shared_realm(m_config);
```

Comment:
This'll create a new Realm with a frozen scheduler. Realm::is_frozen() will return false because it doesn't have a version (and the assert won't fail because it's not in a read transaction), but doing anything which requires a working scheduler with the result will break.

Reply:
Hmm.... it would seem there are aspects of OS I still haven't grasped :-)

Reply:
Turns out not to be needed, so this method will simply be removed